### ISSUE TITLE
Do not open web link with `--browser` if the job failed immediately

### ISSUE THREAD

**Issue by ayushkovskiy** [neuromation/platform-client-python#1084]
```
[ay@archlinux ~]$ neuro run --detach -s cpu-small --browse ubuntu "fail"
Job ID: job-41cab009-ef0f-4712-9cfe-2fbaa632f0a2 Status: pending
Http URL: https://job-41cab009-ef0f-4712-9cfe-2fbaa632f0a2.jobs-dev.neu.ro
Shortcuts:
  neuro status job-41cab009-ef0f-4712-9cfe-2fbaa632f0a2  # check job status
  neuro logs job-41cab009-ef0f-4712-9cfe-2fbaa632f0a2    # monitor job stdout
  neuro top job-41cab009-ef0f-4712-9cfe-2fbaa632f0a2     # display real-time job telemetry
  neuro kill job-41cab009-ef0f-4712-9cfe-2fbaa632f0a2    # kill job
Status: pending Initializing
Status: pending ContainerCreating
Status: failed ContainerCannotRun (container_linux.go:247: starting container process caused "exec: \"fail\": executable file not found in $PATH" )
/home/ay/.pyenv/versions/3.6.3-debug/lib/python3.6/subprocess.py:766: ResourceWarning: subprocess 425302 is still running
  ResourceWarning, source=self)
/home/ay/.pyenv/versions/3.6.3-debug/lib/python3.6/asyncio/base_events.py:510: ResourceWarning: unclosed event loop <_UnixSelectorEventLoop running=False closed=False debug=True>
  source=self)
[ay@archlinux ~]$ [425369:425386:0926/182452.553247:ERROR:browser_process_sub_thread.cc(203)] Waited 3 ms for network service
Opening in existing browser session.
```

Same error happens on errors like `ClusterScaleUpFailed`. And the opened browser link confuses the user.

**Comment by asvetlov** [comment]
Good catch!
Would you provide a fix? Looks like an easy issue.

**Comment by shagren** [comment]
Already fixed in #1059
https://github.com/neuromation/platform-client-python/blob/e35b17a99304ea07f99e05f766f60fa66e1d97f4/neuromation/cli/job.py#L909
